src/Javis.jl

```diff
@@ -69,6 +72,52 @@ abstract type InternalTransition end
 
 abstract type ActionType end
 
+"""
+    SubAction <: ActionType
+
+Defines a subaction with relative frames and a function.
+
+# Fields
+- `frames::UnitRange{Int}`: the frames relative to the parent [`Action`](@ref)
+- `func::Function`: the function that gets called in each of those frames. Takes the following arguments: video, action, subaction, rel_frame
```

Comment:
Could you make sure this line is under or just at 92 characters? https://github.com/invenia/BlueStyle#vs-code-settings